src/truncated/exponential.jl

```diff
@@ -6,8 +6,8 @@ function mean(d::Truncated{<:Exponential,Continuous})
     θ = d.untruncated.θ
     l, r = extrema(d)           # l is always finite
     if isfinite(r)
-        if l == r
-            r
+        if isapprox(l, r, rtol = √eps() * θ, atol = iszero(l) ? √eps() : 0.0)
```

Comment:
Could you add a comment regarding the interplay between ˋrtolˋ and ˋatolˋ here?

Reply:
I may have been overcomplicating it; I simplified and added a comment.